tests/test_special_orthogonal_group.py

```diff
@@ -20,6 +21,8 @@ class TestSpecialOrthogonalGroupMethods(unittest.TestCase):
     _multiprocess_can_split_ = True
 
     def setUp(self):
+        warnings.simplefilter('ignore', category=ImportWarning)
```

Comment:
is this necessary?

Reply:
yes, a lot of annoying warnings otherwise.